apps/api/src/app/inbox/e2e/mark-notification-as.e2e.ts

```diff
@@ -31,10 +31,10 @@ describe('Mark Notification As - /inbox/notifications/:id/mark-as-{status} (PATC
     status,
   }: {
     id: string;
-    status: 'read' | 'unread' | 'archived' | 'unarchived';
+    status: 'read' | 'unread' | 'archive' | 'unarchive';
```

Comment:
Doesn't "archived" and "unarchived" make more sense in "Mark as" here?

Reply:
with the REST convention, these should be verb actions in the present tense